src/components/header/HeaderRight.tsx

```
@@ -6,15 +6,15 @@ import MembershipButton from "../button/MembershipButton";
-import { getToken } from "@pages/socialLogin/utils/token";
+import { getUserToken } from "@pages/socialLogin/utils/token";
 
 interface HeaderRightProps {
   handleHomeMemberInfoToggle: () => void;
   showMemberInfo: boolean;
 }
 export default function HeaderRight(props: HeaderRightProps) {
   const { handleHomeMemberInfoToggle, showMemberInfo } = props;
-  const token = getToken();
+  const token = getUserToken();
   const getNameResponse = token ? useGetName() : null;
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)